src/windows/BluetoothLePlugin.js

```diff
@@ -137,20 +145,14 @@ module.exports = {
 
   startScan: function (successCallback, errorCallback, params) {
 
-    var DeviceInformation = Windows.Devices.Enumeration.DeviceInformation;
-    var DeviceWatcherStatus = Windows.Devices.Enumeration.DeviceWatcherStatus;
-
-    var NAME_KEY = "System.ItemNameDisplay";
-    var RSSI_KEY = "System.Devices.Aep.SignalStrength";
-    var CONTAINER_ID_KEY = "System.Devices.Aep.ContainerId";
-
-    // Drop cache first
-    WATCH_CACHE = {};
-
     if (!WATCHER) {
       // watch BLE devices using device watcher.
-      var selector = Windows.Devices.Bluetooth.BluetoothLEDevice.getDeviceSelectorFromConnectionStatus(false);
-      WATCHER = DeviceInformation.createWatcher(selector, [NAME_KEY, RSSI_KEY, CONTAINER_ID_KEY]);
+      var selector = WindowsBluetooth.BluetoothLEDevice.getDeviceSelectorFromConnectionStatus(false);
+      if (params && params[0] && params[0].isConnectable) {
+        selector += ' AND System.Devices.Aep.Bluetooth.Le.IsConnectable:=System.StructuredQueryType.Boolean#True';
+      }
+      console.info('selector used is', selector);
```

Comment:
Can you remove this?

Reply:
I removed this and a bunch of other logging.